cli/src/main.rs

```diff
@@ -62,6 +62,13 @@ fn run() -> Result<(), CliError> {
                 (@arg quiet: -q --quiet "do not display output")
             )
         )
+        (@subcommand health =>
+            (about: "Splinter commands for inspecting network integrety and health")
+            (@subcommand status =>
+                (about: "Check status of node")
```

Comment:
The description seems too terse. How about clarifying the specific information that `status` displays?

For example:
"shows a node's connected peers, {status?} endpoint, node ID, and version {of what?}"

Or, with more details:  
"queries the status endpoint for a node's connected peers, node ID, and other status information"

_Style note: The first word should start with a lower-case letter and end with "s" to be consistent with the other descriptions (keygen & keyregistry say "generates")._